app/src/main/res/values/color_defs.xml

```diff
@@ -146,4 +145,13 @@
   <color name="admin_highlighted">#A5D3EC</color>
   <color name="coming_soon_end">#674172</color>
   <color name="coming_soon_start">#7659B6</color>
+  <!--DARK MODE-->
+  <color name="oppia_subtext_white">#C4C4C4</color>
```

Comment:
'subtext' feels like a palette detail, ditto for background & container. Can we describe the color directly in its name, instead?

Reply:
Done.